pkg/downloader/manager_test.go

```diff
@@ -77,15 +77,15 @@ func TestFindChartURL(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	name := "alpine"
-	version := "0.1.0"
-	repoURL := "http://example.com/charts"
```

Comment:
Can you please write a new test case alongside the existing test case?

It's important that existing test cases are left intact and are not re-written. There could be other code paths this test case was intended to test for, and by rewriting it you could potentially break that old test.

Reply:
fix.